instrumentation/httpd/opentelemetry.conf

```diff
@@ -32,4 +32,7 @@ OpenTelemetryPath /tmp/output-spans
 
 # OpenTelemetryIgnoreInbound off
 
+# OpenTelemetrySetResource service.name apache-web-server
+# OpenTelemetrySetAttribute foo bar
```

Comment:
Are these attributes global, for all routes? Is it possible to set attributes for specific route?

With nginx it's possible to set these per server, route basis.

Reply:
At the moment those are global settings.